### DISCUSSION TITLE
Plain bg vertical example produces unexpected result

### DISCUSSION THREAD

**Discussion by StEvUgnIn** [marp-team/marp#79] [Q&A] [answered]
```
![bg vertical](https://fakeimg.pl/800x600/0288d1/fff/?text=A)
![bg](https://fakeimg.pl/800x600/02669d/fff/?text=B)
![bg](https://fakeimg.pl/800x600/67b8e3/fff/?text=C)
```

![Capture d’écran 2021-03-01 155742](https://user-images.githubusercontent.com/7686916/109515103-0233ac00-7aa7-11eb-9131-3f102ecf6463.jpg)

edit:
Problem solved:
> Yes, it was due to custom CSS rule: column-count: 2;. I removed it and it works now.

**Comment by yhatt** [comment]
I could not reproduce the glitched view like that. Which OS/browser do you use to browse Marp presentation?

**Reply by StEvUgnIn** [reply]
Yes, it was due to custom CSS rule: `column-count: 2;`. I removed it and it works now.

**Reply by yhatt** [reply]
This may show a bug of advanced backgrounds handled by Marpit framework. Any CSS properties such as `column-count` shouldn't have to make an effect on background images.

**Reply by yhatt** [reply]
UPDATE: It will fix in marp-team/marpit#283.

**Reply by StEvUgnIn** [reply]
Awesome! Sharp fix

**Comment by yhatt** [comment, marked as answer]
**[FIXED]** This problem is solved in these version:

- Marpit >= v2.0.0 (marp-team/marpit#283)
- Marp Core >= v2.0.0 (Depending to Marpit v2.0.0)
- Marp CLI >= v1.0.0 (Depending to Marp Core v2.0.0)
- Marp for VS Code >= v0.18.0 (Depending to Marp Core v2.0.1 and Marp CLI v1.0.1)
